tests/integration/test_schema.py

```diff
@@ -2414,15 +2409,15 @@ async def test_invalid_schema_should_provide_good_error_messages(registry_async_
     subject_name_factory = create_subject_name_factory("test_schema_subject_post_invalid_data")
     test_subject = subject_name_factory()
 
-    schema_str = ujson.dumps({"type": "string"})
+    schema_str = json.dumps({"type": "string"})
     res = await registry_async_client.post(
         f"subjects/{test_subject}/versions",
         json={"schema": schema_str[:-1]},
     )
-    assert res.json()["message"] == "Invalid AVRO schema. Error: Expecting ',' delimiter: line 1 column 17 (char 16)"
+    assert res.json()["message"] == "Invalid AVRO schema. Error: Expecting ',' delimiter: line 1 column 18 (char 17)"
```

Comment:
Interesting change.... But I guess this doesn't matter?

Reply:
https://github.com/aiven/karapace/pull/330/files#diff-7ce48b7ba6aaf19c9eb9f548cd2fe0c0dad95c5a7d0957147477ed5ada247d1bL2317
The deepcopy optimization which used `ujson` introduced the change in the message. So this is reverting it back.